code/modules/emotes/definitions/audible.dm

```diff
@@ -3,10 +3,40 @@
 	emote_message_3p = "USER burps."
 	message_type = AUDIBLE_MESSAGE
 	var/emote_sound
+//Urist add
+	var/gendered
+	var/male_sound //Give these strings for use through the sound.dm sound lists.
+	var/female_sound
+
+//Audible emotes with sound.
+client/
+	var/sound_cooldown = 0 //Can we do an emote with sound?
 
 /decl/emote/audible/do_extra(var/atom/user)
+	var/client/C = user.get_client()
+	if(isnull(C))
```

Comment:
This call is overly-specific; if(!C) achieves the same thing and other edge-cases AFAIK.

Reply:
Handled.